src/Event/EventTest/APITests.cs

```diff
@@ -0,0 +1,81 @@
+using API.Abstraction.Helpers;
+using DAL.Events;
+using EventAPI.Controllers;
+using EventApplication.Entities;
+using EventApplication.Queries;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Routing;
+using Moq;
+using TestsCore;
+
+namespace EventTest
+{
+	public class APITests
+	{
+		[Theory]
+		[InlineData(GetValuesSuites.Empty)]
+		[InlineData(GetValuesSuites.OneValue)]
+		[InlineData(GetValuesSuites.ThreeValues)]
+		[InlineData(GetValuesSuites.ManyValues)]
```

Comment:
What is the logical difference between three and many?

Reply:
Not much